website/docs/r/one_dashboard_json.html.markdown

```diff
@@ -107,6 +107,396 @@ resource "newrelic_one_dashboard_json" "bar" {
   "variables": []
 }
 ```
+### Dashboard With Multiple Pages
+
+The following example demonstrates creating a dashboard with multiple pages, with each page comprising a couple of widgets.
+
+`dashboard.json`
+```json
+{
+  "name": "Multi - Page Dashboard Sample",
+  "description": "An example that demonstrates creating a dashboard with multiple widgets, across a couple of pages.",
+  "permissions": "PUBLIC_READ_WRITE",
+  "pages": [
+    {
+      "name": "Memory Metrics",
+      "description": "Widgets displaying metrics on memory utilization.",
+      "widgets": [
+        {
+          "title": "Memory Utilization",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [
+                  {Account-IDs}
```

Comment:
Can you change this to `account_id` in all examples? That way it's persistent with all examples and Terraform.

Reply:
Replaced all instances of `{Account-IDs}` with `account_id` in the examples.